include/sway/tree/root.h

```diff
@@ -31,7 +31,7 @@ struct sway_root {
 	list_t *scratchpad; // struct sway_container
 
 	// For when there's no connected outputs
-	struct sway_output *noop_output;
+	struct sway_output *headless_output;
```

Comment:
Hm. Sway now supports two types of headless outputs: this one used as a fallback in case there's no other output plugged in, and the ones created via `swaymsg create_output`, which behave just like regular outputs and can be used with e.g. VNC.

Maybe we can rename this one to `fallback_output` to avoid the confusion?

Reply:
Done!